defold-spine/commonsrc/spine_loader.cpp

```diff
@@ -208,6 +209,25 @@ namespace dmSpine
         return &regions[*anim_index];
     }
 
+    static int /*bool*/ loadSequence(dmHashTable64<uint32_t>* name_to_index, spAtlasRegion* atlasRegions, const char *basePath, spSequence *sequence, spAtlasRegion* default_region) {
+        bool is_atlas_available = name_to_index != 0;
+
+        spTextureRegionArray *regions = sequence->regions;
+        char *path = CALLOC(char, strlen(basePath) + sequence->digits + 1);
```

Comment:
While ok, I prefer not dynamically allocating memory if it's easy to avoid.

What are the expected string sizes?
Will a predefined array work?

```c++
char name_buffer[512]; // could be 1024 if you wish
name_buffer[0] = 0; // null terminate it so the spSequence_getPath will work
```

Reply:
But, since I don't really trust their code (there is no way to pass in the length opf the buffer), we'd better go with their approach.

No need for CALLOC though, MALLOC + setting the first character to 0 will suffice. (Calloc clears all elements to 0, whereas we only need the first one to be 0)

Reply:
fixed! thx!